### PR TITLE
Create mysql_version_eol_57.md

### PR DESCRIPTION
**Pull request by rnovikovP**
https://docs.google.com/document/d/1G2KglWSkZBk6Nuo8MWPJZd99zm75z46uo5JZDqUM58I/edit#

### CODE REVIEW

File: docs/advisors/checks/mysql_version_eol_57.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Percona Server and MySQL 5.7 End Of Life
+## Description
+Percona Server and MySQL server 5.7 are declared End Of Life versions. The deadline is on  Oct 21, 2023.
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_version_eol_57.md:3]: I would rephrase this paragraph to:

Percona Server for MySQL Server version 5.7 will reach End-of-life on October 21st, 2023.
Percona does not provide support or issue hotfixes for retired releases.

Comment (catalinaadam) [inline docs/advisors/checks/mysql_version_eol_57.md:3]: Did you mean Percona Server for MySQL Server 5.7?

File: docs/advisors/checks/mysql_version_eol_57.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Percona Server and MySQL 5.7 End Of Life
+## Description
+Percona Server and MySQL server 5.7 are declared End Of Life versions. The deadline is on  Oct 21, 2023. 
+After that date the software will be out of support and users that will still be running that version will not
+get any further update or bug fix.
+
+## Resolution
+Contact Percona to get help to plan the software migration from version 5.7 to 8.0, or to discover Percona extended support for version 5.7.
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_version_eol_57.md:8]:
```suggestion
[Contact Percona](https://www.percona.com/about/contact) to get extended support for version 5.7, or help you plan the migration to version 8.0 (latest).  
```

File: docs/advisors/checks/mysql_version_eol_57.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+https://www.percona.com/about/contact
+
+https://www.percona.com/services/policies/percona-software-support-lifecycle
```
Comment (catalinaadam) [inline docs/advisors/checks/mysql_version_eol_57.md:12]:
```suggestion
For more information, check out the [Percona software support lifecycle information](https://www.percona.com/services/policies/percona-software-support-lifecycle)
```

File: docs/advisors/checks/mysql_version_eol_57.md (hunk 1 above)
Comment (rnovikovP) [reply docs/advisors/checks/mysql_version_eol_57.md:3]: Both of them

Comment (rnovikovP) [reply docs/advisors/checks/mysql_version_eol_57.md:3]: We'll improve this later; as for now, WIP on final messages about 5.7 EOL and our offers on it.

**Comment by rnovikovP** [comment]
This is not planned as the final version, but a version we can release with and in several days to update with better messaging

**Review by catalinaadam — APPROVED** [review]
